setup.md

```diff
@@ -4,23 +4,28 @@ title: Setup
 ---
 
 > ## Data
-> Data for this lesson is from the Portal Project Teaching Database -
+> Data for this lesson is based on the Portal Project Teaching Database -
 > [available on FigShare](https://figshare.com/articles/Portal_Project_Teaching_Database/1314459).
->
-> We will use the six files listed below for the data in this lesson.
-> Download these files to your computer either by clicking
-> [this link](https://github.com/weecology/portal-teachingdb/archive/master.zip),
-> which will give you everything in a single compressed file.
-> You'll need to unzip this file after downloading it.
->
-> Or download each file indvidually with the following links:
+> 
+> For this lesson we will use the eight files listed below for data.
+> These can be download by following the below links
```

Comment:
Could you please rephrase it so that it is easier to read?

Something like this:

> Data (files) used in this lesson are from the Portal Project Teaching Database available on Figshare. Specifically, we use the following eight data files:
>
> - file one
> - file two
> - file three
> ...
Please download them (by clicking on the corresponding links) and move them to the same directory.

Reply:
I agree that its a little difficult to read, but aimed at being correct and inline with our discussion:

"Also, on the setup page we should change the wording to emphasize the fact that these three files are not in the zip archive nor can they be found on Figshare..."

I can rephrase, but should the change still highlight the fact that 
1) some files are not from Portal Project Teaching Database, and 
2) should it still link to the zip file?

Reply:
Feel free to use my suggestion as a base and expand it.
We can provide a zip file ourselves instead of relying on ever-changing files. In this case, we can say that our files are from [location] retrieved on [date].